flake.nix

```diff
@@ -99,14 +102,16 @@
                 cargo-watch
                 osmium-tool
                 geos
+                mold
```

Comment:
you've got to export `RUSTFLAGS="-C link-arg=-fuse-ld=mold"` if you want this to do anything

Reply:
Yep, I know, I could add a `.cargo/config.toml` in the repository, but that would impose this dependency to all developers and most of them don't use `nix` for setup. But I do have this configuration in `~/.cargo/config.toml`... and my full build drop from 1m50s to about 1m10s.

Reply:
We should modify it in the flake.nix : if the flake provides mold then we can be assured that within a `nix develop` shell mold will be available.

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7560/commits/07a9685d6106f76cbc766a70d4eaaaf0f5ba4bf6.